File: erettsegi_emelt/2018_okt/c#/Telek.cs
public struct Telek {
    public readonly bool parosE;
    public readonly int szelesseg;
    public readonly int hazszam;
    public readonly char keritesSzine;

    public Telek(bool parosE, int hazszam, int szelesseg, char keritesSzine) {
        this.parosE = parosE;
        this.szelesseg = szelesseg;
        this.keritesSzine = keritesSzine;
        this.hazszam = hazszam;
    }
}

File: erettsegi_emelt/2018_okt/java/Telek.java
public class Telek{

    public final boolean parosE;
    public final int szelesseg;
    public final int hazszam;
    public final char keritesSzine;

    public Telek(int hazszam, int szelesseg, char keritesSzine) {
        this.parosE = hazszam % 2 == 0;
        this.szelesseg = szelesseg;
        this.keritesSzine = keritesSzine;
        this.hazszam = hazszam;
    }
}